sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -32,6 +47,27 @@ export interface CorrelationFilter {
     userProperties?: any;
 }
 
+// @public
+export type CountDetails = {
```

Comment:
Suggestion: Rename this to `MessageCountDetails` or `EntityMessageCount` ?

Reply:
Updated